README.md

```diff
@@ -74,3 +74,39 @@ loader = tfrecord.tfrecord_loader("/path/to/data.tfrecord", None, {
 for record in loader:
     print(record["label"])
 ```
+
+### Transforming input
+
+There are use cases when you might not want to return your input directly after reading from the tfrecord files.
+A trivial example could be to normalize your input, if you haven't saved the normalized feature in the files.
+
+You can easily achieve this by leveraging the `transform_func` argument.
+There might also be use cases to remove a field after performing a transformation, this can be leveraged using the `removed_fields`.
+
+Both the arguments mentioned above are available in all readers including `tfrecord.tfrecord_loader` and `tfrecord.torch.TFRecordDataset`.
+ 
+```python
+import tfrecord
+
+def transformation(features):
```

Comment:
Change this to an example where you decode an image with opencv. It's more useful.

Reply:
Can you confirm if this looks good and this what you're suggesting? Haven't worked with images in a while, so don't want to assume. 

```python
import tfrecord
import cv2

def transformation(features):
    # get BGR image from bytes
    features["image"] = cv2.imdecode(features["image"], -1)
    return features


description = {
    "image": "bytes",
}

dataset = tfrecord.torch.TFRecordDataset("/path/to/data.tfrecord",
                                         index_path=None,
                                         transform=transformation)
```

One use case I find extremely beneficial is that padding variable length sequences, so that dataloader works out of the box.

Reply:
Looks good. Just call transformation, decode_image for clarity.